docs/sdks/server-sdks/dotnet.md

```diff
@@ -3,39 +3,65 @@ import TabItem from '@theme/TabItem';
 
 # .NET
 
-Eppo's open source .NET SDK can be used for both feature flagging and experiment assignment:
+Eppo's open source .NET SDK can be used for feature flagging and experiment assignment and Multi-Armed Contextual Bandits:
 
 - [GitHub repository](https://github.com/Eppo-exp/dot-net-server-sdk)
 - [Package](https://www.nuget.org/packages/Eppo.Sdk)
+- [Nuget Package](https://www.nuget.org/packages/Eppo.Sdk)
 
-## 1. Install the SDK
+
+## Getting Started
+
+### Installation
 
 In your .NET application, add the Eppo.Sdk Package from Nuget.
 
-```
+```shell
 dotnet add package Eppo.Sdk
 ```
 
-## 2. Initialize the SDK
+### Usage
 
-Initialize the SDK with a SDK key, which can be generated in the Eppo interface. Initialization should happen when your application starts up to generate a singleton client instance, once per application lifecycle:
+Begin by initializing a singleton instance of Eppo's client with an SDK key from the [Eppo interface](https://eppo.cloud/feature-flags/keys). Once initialized, the client can be used to make assignments anywhere in your app. Initialization should happen when your application starts up to generate a singleton client instance, once per application lifecycle:
 
-Then use that to create the EppoClient instance.
+
+#### Initialize once
 
 ```csharp
-var eppoClientConfig = new EppoClientConfig(apiToken, new AssignmentLogger());
+var eppoClientConfig = new EppoClientConfig("<SDK_KEY>", new AssignmentLogger()); // Don't worry about the `AssignmentLogger` just yet. More on it below.
 var eppoClient = EppoClient.Init(eppoClientConfig);
 ```
 
 After initialization, the SDK begins polling Eppo's API at regular intervals to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments thereafter are effectively instant. For more information, see the [architecture overview](/sdks/architecture/overview) page.
 
 If you are using the SDK for experiment assignments, make sure to pass in an assignment logging callback (see [section](#define-an-assignment-logger-experiment-assignment-only) below).
 
-### Define an assignment logger (experiment assignment only)
+#### Assign anywhere
+
```

Comment:
I think we should have a table of the parameters like we do for Querying for a Bandit Action. For example, it is not clear what that fourth "control" argument is/does.

Reply:
done